db/R/query.base.R

```diff
@@ -70,7 +70,7 @@ query.base.con <- function(settings,...){
 ##' @param con database connection object
 ##' @return nothing, as a side effect closes all open connections
 ##' @author Rob Kooper
-##' @export
+##' @export query.close
```

Comment:
probably fine for now, but the sooner we can migrate names the less of a problem this will be.